pymodbus/client/serial_asyncio/__init__.py

```diff
@@ -18,6 +18,7 @@
 import asyncio
 import os
 
+
```

Comment:
This is an imported package, we do not makes changes in here, unless there is a corresponding pull request in their repo.

The only reason we have it, is because they take a very long time in responding to pull requests (I have filed 2), so it was imported to solve the problem at hand.

We do not do "cosmetic" changes in their source.

This passes without problems in our current CI, so this makes me even more afraid.

Reply:
Yes of course it is disabled, because we do not want that directory touched. precommit must follow what we have already configured.